terminus-ui/popover/src/popover.component.scss

```diff
@@ -0,0 +1,77 @@
+@import './../../scss/helpers/color';
+@import './../../scss/helpers/shadows';
+@import './../../scss/helpers/spacing';
+
+$PLACEMENT_MARGIN: 5px;
+
+.popover__container {
+  @include elevation-element(snackbar);
+  background-color: color(pure);
+  display: none;
+  padding: spacing(default);
+  position: absolute;
+
+  .popover__arrow {
+    border-color: color(pure);
+    border-style: solid;
+    height: 0;
+    position: absolute;
+    width: 0;
+  }
+
+  &[x-placement^='top'] {
+    margin-bottom: $PLACEMENT_MARGIN;
+    .popover__arrow {
```

Comment:
No need to change it, but for future reference we have a mixin to help create these triangles easily